spec/db/seed_development_spec.rb

```diff
@@ -40,11 +40,11 @@
   end
 
   it "addresses are in the db" do
-    expect(Address.all.size).to eq(seed_users-1)
+    expect(Address.all.size).to eq(MembershipApplication.where(state: :accepted).count)
   end
 
   it "companies are in the db" do
-    expect(Company.all.size).to eq(seed_users-1)
+    expect(Company.all.size).to eq(MembershipApplication.where(state: :accepted).count)
```

Comment:
I'm wondering if we need line 5 in this file?  (we override the ENV value anyway).

Reply:
Line 5 is not really necessary, I could just put the literal in line 14. I guess it would somewhat enhance readability. Is that what you mean?

Reply:
Yes, that would be easier to understand if `ENV` and the specific key were in the same statement.